gms/versioned_value.hh

```diff
@@ -142,6 +142,14 @@ public:
             return dht::global_partitioner().to_sstring(*tokens.begin());
         }
 
+        sstring make_cdc_streams_timestamp_string(std::optional<db_clock::time_point> t) {
```

Comment:
It's sad to inject cdc information into gossip. I don't have a proposal to fix, but see if you can think of one.

Reply:
I'm referring you back to the design document, "Roadmap" section:
https://docs.google.com/document/d/1mcxq1uNBDi9TD-0Diud4euK_1-G1Yogv4IHAzDSoNsI/edit#heading=h.qet7l07pc9n

> In the initial version, node responsible for topology change (joining node, node handling nodetool command for commands that target inactive node):
Generates new streams and mappings from vnode + shard to stream and writes it down to a regular (maybe hidden from user) table
Selects a timestamp 3 (or 5) minutes away from the current time (according to its own clock)
Gossips this timestamp to the whole cluster
We assume that every node gets the timestamp by the time the timestamp is active (i.e., before the internal clock of every node reaches this timestamp)
Once a node receives a new timestamp from gossip it reads the table generated in step a. and starts using the new state from a time of timestamp. By new state we mean the whole topology.

I believe you said "Looks good".

Reply:
I'm sure I did, but I don't have that document in my mind, and neither will others who read the code.